src/vs/workbench/contrib/chat/browser/chatAccessibilityProvider.ts

```diff
@@ -37,37 +36,43 @@ export class ChatAccessibilityProvider implements IListAccessibilityProvider<Cha
 		}
 
 		if (isResponseVM(element)) {
-			return this._getLabelWithCodeBlockCount(element);
+			return this._getLabelWithInfo(element);
 		}
 
 		return '';
 	}
 
-	private _getLabelWithCodeBlockCount(element: IChatResponseViewModel): string {
+	private _getLabelWithInfo(element: IChatResponseViewModel): string {
 		const accessibleViewHint = this._accessibleViewService.getOpenAriaHint(AccessibilityVerbositySettingId.Chat);
 		let label: string = '';
-		const fileTreeCount = element.response.value.filter((v) => !('value' in v))?.length ?? 0;
+
+		const toolInvocation = element.response.value.filter(v => v.kind === 'toolInvocation' || v.kind === 'toolInvocationSerialized').filter(v => v.isComplete);
+		let toolInvocationHint = '';
+		if (toolInvocation.length) {
+			toolInvocationHint = localize('toolInvocationsHint', "Action required: {0} ", toolInvocation.map(v => v.invocationMessage.toString()).join(', '));
```

Comment:
This isn't the confirmation message, maybe you want to read `.confirmationMessages`?